scripts/webpack/getResolveAlias.js

```diff
@@ -4,26 +4,48 @@ const findRepoDeps = require('../monorepo/findRepoDeps');
 const findGitRoot = require('../monorepo/findGitRoot');
 const { readConfig } = require('../read-config');
 
+function getOutputPath(packageJson) {
+  for (const mainField of ['module', 'main']) {
+    const main = packageJson[mainField];
+    if (main && main.includes('dist/es')) {
+      return 'dist/es';
+    } else {
+      return 'lib';
+    }
```

Comment:
Nesting is wrong (you're returning `lib` if the field doesn't exist). One possible workaround:
```suggestion
    if (main) {
      return main.includes('dist/es') ? 'dist/es' : 'lib';
    }
```